Desktop/components/JASP/Controls/Form.qml

```diff
@@ -100,7 +100,7 @@ AnalysisForm
 			id:					errorMessagesBox
 			visible:			form.errors !== ""
 			color:				jaspTheme.controlErrorBackgroundColor
-			width:				parent.width
+			width:				form.implicitWidth
```

Comment:
Why do you change it to implicitWidth ?

Reply:
Because the width of the parent changes when the scrollbar appears and dissappears. I figured this would give it a bit more stable sizing (and avoid some property-update-loops).

Not 100% it is necessary but yeah...